.github/pr-custom-review.yml

```diff
@@ -4,15 +4,11 @@ rules:
     check_type: changed_files
```

Comment:
This entire file isn't required in Substrate? We don't have any production network runtime config in Substrate and we already use github to enforce 2 approvals for any pr.

Reply:
Or is there any other usage I'm not aware of?

Reply:
Only teams belonging to **matched** rules will be requested for reviews, e.g. if the PR does not change any 🔒 line, then the locks-review-team's review will not be requested.

The "Substrate developers" rule matches `.*` for `changed_files` which means its teams will be requested for every pull request that changes any file. How those teams are notified is configurable according to https://docs.github.com/en/organizations/organizing-members-into-teams/managing-code-review-settings-for-your-team#about-team-notifications (https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/about-notifications#reasons-for-receiving-notifications might also be of interest).

Reply:
Hmmm. Why do we need to ping everybody for every pr? That doesn't sound good. Can we please change that?

Reply:
> Why do we need to ping everybody for every pr?

You will not *have to* according to https://docs.github.com/en/organizations/organizing-members-into-teams/managing-code-review-settings-for-your-team#configuring-auto-assignment; an organization admin should balance the team reviews to specific people at a time instead of everyone ("Load balance", "Round robin", ...). cc @TriplEight 

This feature can't be implemented in pr-custom-review as it is right now because it does not have a persistent storage for tracking who would be assigned to which PR.

Reply:
> Why can check-reviews not just ensure that 2 people of this group have approved the pr.

That is already how it works.

The feature I'm judging currently unfeasible in the comment before are "Round robin" or "Load balance" assignments that GitHub does (that is when the persistent storage *would* come in).

Reply:
Ahh okay. Ty for explaining this! :)